.azure-pipelines/gpu-benchmark.yml

```diff
@@ -1,3 +1,7 @@
+trigger: none
```

Comment:
Trigger yes on each each commit to the master

Reply:
Should I remove the `trigger` command?